Makefile

```diff
@@ -70,4 +85,4 @@ clean-all: clean clean-gentype
 
 .DEFAULT_GOAL := build
 
-.PHONY: build watch ninja bench dce test test-syntax test-syntax-roundtrip test-gentype test-all lib artifacts format checkformat clean-gentype clean clean-all
+.PHONY: build watch ninja bench dce test test-syntax test-syntax-roundtrip test-gentype test-all lib playground playground-cmijs test-playground playground-release artifacts format checkformat clean-gentype clean clean-all
```

Comment:
There is no `test-playground` defined in the Makefile?

Reply:
removed.